resources/public/index.html

```diff
@@ -71,8 +71,17 @@
     </div>
 
     <script src="//cdnjs.cloudflare.com/ajax/libs/react/0.9.0/react.min.js"></script>
+
+    <!-- DEV version -->
+    <script src="js/out-dev/goog/base.js" type="text/javascript"></script>
+    <script src="js/cloujera-dev.js" type="text/javascript"></script>
+    <!-- end DEV version -->
+
+    <!-- PROD version
```

Comment:
this still needs to be commented out by hand to deploy to prod. I'm talking with one of the guys at hacker school tomorrw, he knows Om/cljs and should know what to do.